konig-schemagen-maven-plugin/src/main/java/io/konig/maven/KonigSchemagenMojo.java

```diff
@@ -1573,13 +1573,15 @@ private void configureBigQueryTransform() throws BeamTransformGenerationExceptio
 		
 		List<ShowlNodeShape> nodeList = listingConsumer.getList();
 		File projectDir = new File(mavenProject.getBasedir(), "target/generated/beam");
+		GoogleCloudService googleService = new GoogleCloudService();
 		if (!nodeList.isEmpty()) {
 			BeamTransformRequest request = BeamTransformRequest.builder()
 				.groupId(mavenProject.getGroupId())
 				.artifactBaseId(mavenProject.getArtifactId())
 				.version(mavenProject.getVersion())
 				.projectDir(projectDir)
 				.nodeList(nodeList)
+				.gcpProjectId(googleService.getProjectId())
```

Comment:
The GCP project Id is environment specific.  This won't work.  Let's discuss during our scrum meeting.